### DISCUSSION TITLE
Using Paperless-ng for an existing network PDF library

### DISCUSSION THREAD

**Discussion by avnibu** [jonaswinkler/paperless-ng#961] [General]
Hello all,

I would like to test paperless-ng as a document index on an existing (Windows) network PDF library. 

I have a 10000+ PDF documents (reference documents) already organized in folders on a Windows Network Share that I would like to be able to search within. Can Paperless-ng be setup to work on a setup like this? 

Thanks in advance for your answers...

**Comment by jonaswinkler** [comment]
At this point, paperless-ng is unable to re-use an already existing folder structure.
